Lombiq.Tests.UI/Lombiq.Tests.UI.csproj

```diff
@@ -59,11 +59,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Atata" Version="2.9.0" />
-    <PackageReference Include="Atata.Bootstrap" Version="2.1.0" />
+    <PackageReference Include="Atata" Version="2.14.1" />
+    <PackageReference Include="Atata.Bootstrap" Version="2.2.0" />
     <PackageReference Include="Atata.HtmlValidation" Version="2.4.0" />
```

Comment:
This is intentionally not upgraded because it would introduce more html-validation warnings mainly probably coming from OC source.